charts/influxdb/values.yaml

```diff
@@ -240,22 +239,56 @@ initScripts:
     init.iql: |+
       CREATE DATABASE "telegraf" WITH DURATION 30d REPLICATION 1 NAME "rp_30d"
 
+
+## InfluxDB backup parameters
+##
 backup:
   enabled: false
-  schedule: "0 0 * * *"
-  annotations: {}
-  podAnnotations: {}
-
-  ## Google Cloud Storage
-  # gcs:
-  #    serviceAccountSecret: influxdb-backup-key
-  #    serviceAccountSecretKey: key.json
-  #    destination: gs://bucket/influxdb
-
-  ## Azure
-  ## Secret is expected to have connection string stored in `connection-string` field
-  ## Existing container will be used or private one withing storage account will be created.
-  # azure:
-  #   storageAccountSecret: influxdb-backup-azure-key
-  #   destination_container: influxdb-container
-  #   destination_path: ""
+  ## Directory where backups are stored
+  ##
+  directory: "/backups"
+  ## Retention time in days for backups
+  ##
+  retentionDays: 10
+  ## Cronjob configuration
+  ## This cronjob is used to create InfluxDB backups
+  ##
+  cronjob:
+    ## Schedule in Cron format to save snapshots
+    ## See https://en.wikipedia.org/wiki/Cron
+    ##
+    schedule: "0 2 * * *"
+    ## Number of successful finished jobs to retain
+    ##
+    historyLimit: 3
+    ## Pod annotations
+    ## ref: https://kubernetes.io/docs/concepts/overview/working-with-objects/annotations/
+    ##
+    podAnnotations: {}
+  ## Storage providers where to upload backups
+  ##
```

Comment:
Please bring these values one level up directly nest under `backup` instead of nesting in `cronjob`.

Reply:
why?

Reply:
The same reason we don't have a `statefulset:` value, whatever that is :)

Reply:
also needless breaking changes